### DISCUSSION TITLE
how can define profile page

### DISCUSSION THREAD

**Discussion by webparadoxer** [filamentphp/filament#776] [Help] [answered]
How can i do page with sidebar link going to  with parametr auth()->id()             
`'edit' => Pages\EditUser::route('/{record}/edit'),
`

**Comment by danharrin** [comment, marked as answer]
Instead of using a resource, create a custom page and use the form builder as described in the documentation [here](https://filamentadmin.com/docs/forms/getting-started).

**Comment by lucacalcaterra** [comment]
It's still vallid ? How I would like to implement a filament page for user profile. How can i reuse an EditRecord page without resource ? Thanks

**Reply by danharrin** [reply]
Just follow the form builder docs, it's very simple. You'll spend more time trying to get the EditRecord page working than just setting up the form builder on a custom page.

We also have profile plugins available - https://github.com/ryangjchandler/filament-profile, https://github.com/jeffgreco13/filament-breezy

**Reply by lucacalcaterra** [reply]
You have right :-) Thanks ! Very useful...
